### DISCUSSION TITLE
Your account has been flagged

### DISCUSSION THREAD

**Discussion by JhnClb** [community/community#33494] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/113907798/191016365-49233b1e-b77e-4e1d-9c83-ad8c215ffaf2.png)
Good evening, I'd like to ask you guys on how to resolve this problem, I accidentally mistyped gmail as gmai, so when I tried to change it, it put me as " Your account has been flagged"..

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Jhnclb! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by QIN2DIM** [comment]
I did so, but technical support did not give me an answer, and I was very disappointed that more than twenty days had passed since the opening of the ticket.

**Reply by 5diejguapf** [reply]
Does you have answer from support?

**Reply by Ghislain1** [reply]
Even me i dont get answer from support since 1 Month left.

**Reply by QIN2DIM** [reply]
I got unblocked the day after posting this... wtf

**Reply by MuhammadHuzaifa881** [reply]
Your account has been flagged. How i fix this? 
they dont answer me after i get the ticket since 1 month

**Comment by siljemogster** [comment]
"Your account has been flagged." How can I fix this?

**Comment by armaniJPrypin21** [comment]
arent email provider recognizing errors such taht? 😉

**Comment by armaniJPrypin21** [comment]
try to install SocialEngeneeringToolkit and spoof 😂

**Comment by hamidsh07** [comment]
I did so, but technical support did not give me an answer, and I was very disappointed that more than twenty days had passed since the opening of the ticket.

**Comment by krona12** [comment]
I don’t understand what happened to my account, but technical support did not give me an answer even seven days passed after I commit my ticket, which is absurd

**Reply by love-yuri** [reply]
Have you solved this problem? Mine is the same, it's been 8 days and I still haven't received a response!

**Comment by bashangbasir** [comment]
Hi , 
Same as me. Already submitted support ticket to Github but no response from them. It had been 2 weeks for my case.

**Comment by queenofcorgis** [comment]
Thank you for your patience to those who have submitted tickets. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
